PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+
+# What you've done
+...
+
+# Why have you done it
+...
+
+# Screenshot (if visual)
+...
+
+# Any testing carried out
+...
```

Comment:
+1. Usually I've seen people follow these details. Appreciate suggestions.

1. PR title should be at most 50 chars. More than 50 smells of bad description.
2. PR title should be in the future tense (not sure if I got that correctly). ex: `Fixing ...` and `Fixed ...` are wrong. Should be `Fix ...`.
3. Title shouldn't contain references to issue numbers. That should be handled in the description.